components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -153,10 +164,18 @@ const ContributorProfileHeader = ({
             size={120}
             isCircle
           />
+          <Link
+            href={cardPageUrl(username!)}
+            className="absolute bottom-0 z-10 grid w-10 h-10 rounded-full shadow-md right-1 place-content-center border-conic-gradient"
```

Comment:
can we change width/height to "w-11 h-11" so that the outside ring looks a bit thinner?

Reply:
Update: I just made the requested adjustments on mobile @isabensusan